tests/regionizers/test_voronoi_regionizer.py

```diff
@@ -11,7 +11,7 @@
 
 def test_empty_gdf_value_error(gdf_empty) -> None:  # type: ignore
     """Test checks if empty GeoDataFrames are disallowed."""
-    with pytest.raises(ValueError):
+    with pytest.raises(AttributeError):
```

Comment:
Why like that, if you raise `ValueError` when too few seeds are passed?

Reply:
Also wondering

Reply:
It fails on first line of code
`seeds_wgs84 = seeds.to_crs(epsg=4326)`
that's why it has been changed to AttributeError - missing geometry column.